src/Joystick/Joystick.cc

```diff
@@ -113,6 +113,8 @@ Joystick::Joystick(const QString& name, int axisCount, int buttonCount, int hatC
     , _multiVehicleManager  (multiVehicleManager)
     , _customActionManager  (qgcApp()->toolbox()->settingsManager()->customMavlinkActionsSettings()->joystickActionsFile())
 {
+    qCDebug(JoystickLog) << Q_FUNC_INFO << this;
```

Comment:
Should these remain in the production code? Seems more like useful for detailed debugging of a specific scenario as opposed to generic logging. Maybe keep in commented out for usage as needed?

Reply:
Sure, I usually just use it to verify correct usage of QObject parenting cleanup